server/src/main/kotlin/com/kamelia/hedera/database/Connection.kt

```diff
@@ -23,6 +24,15 @@ object Connection {
 
     private fun hikari(): HikariDataSource {
         val config = HikariConfig("/hikari.properties")
+
+        if (Environment.isProd) {
+            config.apply {
+                jdbcUrl = "jdbc:postgresql://${Environment.databaseHost}:${Environment.databasePort}/${Environment.databaseName}"
+                username = Environment.databaseUsername
+                password = Environment.databasePassword
+            }
+        }
+
```

Comment:
erh, not sure about loading a properties file at all. I think since the config is at least partially loaded here, you should go all the way

Reply:
We need to load it in tests context since it uses an embedded database, with all sorts of fancy settings we're not going to extract in environment variables. We may create a `TEST` env to handle this case properly?

Reply:
mmh, more simply, we could use the dev env to always load the config file

```suggestion
        val config = if (Environment.isProd) {
            HikariConfig().apply {
                jdbcUrl = "jdbc:postgresql://${Environment.databaseHost}:${Environment.databasePort}/${Environment.databaseName}"
                username = Environment.databaseUsername
                password = Environment.databasePassword
            }
        } else {
            HikariConfig("/hikari.properties")
        }
```

this way, we have default settings for the db both in dev and tests

Reply:
Yup, definitely agree with you!